apps/landing/features/home/HeadingSection/SignUpModal/FormInput.tsx

```diff
@@ -28,29 +29,46 @@ const FormInput: FC<Props> = ({
 	onBlur,
 	prefix,
 }) => {
-	const borderOpacity = useSharedValue(0);
+	const borderColor = useSharedValue('transparent');
+
 	const handleHoverIn = () => {
-		borderOpacity.value = borderOpacity.value === 1 ? 1 : 0.7;
+		borderColor.value = '#566674';
 	};
+
 	const handleHoverOut = () => {
-		borderOpacity.value = borderOpacity.value === 1 ? 1 : 0;
+		console.log('hover out: ', error);
+		if (error !== '') borderColor.value = '#E34237';
```

Comment:
`if (error !== '')` might be change to if (error)

Reply:
> `if (error !== '')` might be change to if (error)

I think `if (error)` is not the same as `if (error = '')`

Reply:
with boolean conversion, '' mean `False`, so with non-empty string with return `True`